src/WebJobs.Script/Description/BindingMetadata.cs

```diff
@@ -3,14 +3,37 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Description
 {
+    /// <summary>
+    /// Represents the metadata for a function parameter binding.
+    /// </summary>
     public class BindingMetadata
     {
+        /// <summary>
+        /// Gets or sets the name of the binding.
+        /// </summary>
         public string Name { get; set; }
 
+        /// <summary>
+        /// Gets or sets an app setting name for a setting whose value
+        /// contains the connection details that should be used for this binding.
+        /// E.g. for a Queue binding, this might be set to the name of a an app
+        /// setting containing the Azure Storage connection string to use.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
Yeah, my plan is to make the rest of the bindings consistent, in a follow up immediately after this checkin.